sources/api/schnauzer/src/lib.rs

```diff
@@ -122,6 +122,7 @@ pub fn build_template_registry() -> Result<handlebars::Handlebars<'static>> {
     template_registry.register_helper("join_map", Box::new(helpers::join_map));
     template_registry.register_helper("default", Box::new(helpers::default));
     template_registry.register_helper("ecr-prefix", Box::new(helpers::ecr_prefix));
+    template_registry.register_helper("host-and-port", Box::new(helpers::host_and_port));
```

Comment:
My `ecr-prefix` unfortunately failed to observe the naming convention started by `join_map`, and the long standing metricdog PR will continue with the underscore convention instead of snake case.

```suggestion
    template_registry.register_helper("host_and_port", Box::new(helpers::host_and_port));
```

Reply:
Now the helper is just called `host`